src/ch/imedias/rsccfx/view/RsccSupportView.java

```diff
@@ -81,23 +81,23 @@ private void layoutForm() {
     // this.setPadding(new Insets(5, 25, 5, 25)); // TODO: set paddings for "center"
```

Comment:
This and the next line as well, Comments still needed? Maybe they are not from you, then you can leave them in